### DISCUSSION TITLE
Spegnimento controllato MINI PC con docker compose su linux debian

### DISCUSSION THREAD

**Discussion by CesareDebian** [indomus/forum#3276] [HUB: Home Assistant]
Salve,
sto seguendo la guida per effettuare lo spegnimento controllato del Mini PC in seguito all'avvicinarsi allo 0% della batteria residua dell'UPS.
Ho provato a creare il file shell_comand.yaml, ma ogni tentativo è risultato errato ( errore 127 ).
C'è qualcuno che è riuscito a risolvere questo problema?
Potrebbe riportare i passaggi da effettuare per generare questa automazione?
Grazie anticipatamente.

**Comment by snakuzzo** [comment]
1. non scrivere i titoli delle discussioni in maiuscolo. Non si fa! Quindi correggi altrimenti eliminiamo la discussione
2. chi o cosa ti dà errore 127 ? Non si capisce una fava se la spieghi così
3. Riporta esattamente quello che hai fatto qui sul forum e usando tassativamente il markdown per scrivere il codice

**Reply by CesareDebian** [reply]
Scusa per il titolo! Provvedo subito!

**Reply by CesareDebian** [reply]
Ho creato un file shell_command.yaml ed ho scritto:
`shutdown_minipc: ssh -l deb 192….."sudo shutdown`
e poi all’interno del file scripts.yaml ho inserito:
``` yaml
shutdown_minipc:
    alias: Shutdown_Mini-PC Script
    sequence:
      - service: shell_command.shutdown_minipc
```
 Ottenendo l’errore 127.

L’errore 127 l’ho ottenuto anche cambiando deb con homeassistant.

Inoltre, ho ottenuto lo stesso errore 127 anche provando ad inserire nel file shell_command.yaml:

`shutdown_deb_cmd: '/usr/bin/sudo /sbin/shutdown now`

**Comment by CesareDebian** [comment]
Ho notato che in un’altra discussione si invitava ad osservare questo link:
https://github.com/indomus/forum/discussions/1966#discussioncomment-3857411
Purtroppo non riesco a capire come risolvere il problema. La discussione è forse sospesa?
